Tests/MapboxMapsTests/Style/Generated/Layers/SymbolLayerTests.swift

```diff
@@ -84,7 +84,7 @@ class SymbolLayerTests: XCTestCase {
        layer.layout?.symbolZOrder = Value<SymbolZOrder>.testConstantValue()
        layer.layout?.textAllowOverlap = Value<Bool>.testConstantValue()
        layer.layout?.textAnchor = Value<TextAnchor>.testConstantValue()
-       layer.layout?.textField = Value<String>.testConstantValue()
+       layer.layout?.textField = Value<Formatted>.testConstantValue()
```

Comment:
you will need to add this property as a static property to `Value<Formatted>` once @neelmistry94  adds the `Formatted` enum.